package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ceramic-observability",
   "version": "1.0.3",
-  "private": false,
```

Comment:
this is a public package?  tho we only care about publishing the service-metrics one

Reply:
It is :) Not private means public.